### DISCUSSION TITLE
Wakewords/microphone

### DISCUSSION THREAD

**Discussion by thorin29** [timmo001/home-assistant-assist-desktop#122] [Q&A] [answered]
I have this installed on Ubuntu, are wake words and STT supported with this project? If so, I do not see anywhere to configure microphone settings.

FYI, I have the wyoming protocol installed with all dependencies. Wake word and STT detection working on android with Home Assist (separate project but Home Assistant assist is working).

**Comment by timmo001** [comment, marked as answer]
Speech to text is, but no wake words. I would love to add them somehow though.

`Alt+Shift+A` to trigger voice, or use the microphone icon.

**Comment by thorin29** [comment]
When I click the microphone icon it does not capture my speech. Do I need a local speech to text or does the faster-whisper STT on HA get used?
